internal/controller/atlasschema_controller.go

```diff
@@ -164,7 +164,7 @@ func (r *AtlasSchemaReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		_, err = wd.WriteFile("atlas.hcl", buf.Bytes())
 		return err
 	}
-	cli, err := r.atlasClient(wd.Path(), data.Cloud)
+	cli, err := r.atlasClient(wd.Path(), nil)
```

Comment:
Why?

Reply:
if "ATLAS_TOKEN" is defined, the logic will fail before going to `WhoAmI` command